independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus-extension/code/quarkiverse/java/.github/dependabot.yml

```diff
@@ -9,3 +9,6 @@ updates:
     directory: "/" # Location of package manifests
     schedule:
       interval: "daily"
+    ignore:
+      # don't update Quarkus as we want the extension to depend on the minimum working version, thus allowing for inclusion of the extension in tooling for older Quarkus versions
+      - dependency-name: io.quarkus:*
```

Comment:
shouldn't it still be okey to get micro updates within a stream ?

Reply:
I don't think so, because the API won't change, which is what the extension depends on to build

Reply:
"because the API ~~won't~~ SHOULD NOT change" :smile: